### DISCUSSION TITLE
Ability to customize shortcuts

### DISCUSSION THREAD

**Discussion by japanese-goblinn** [intellibar/main#17] [Ideas]
This is probably in your plans already, but I still want to document this request.

**Comment by astoilkov** [comment]
Do you mean customizing all shortcuts (not only the one for opening IntelliBar)? If this is the case, this isn't yet on the roadmap.

**Reply by japanese-goblinn** [reply]
Oh! I didn't realize that these shortcuts can be edited. Perhaps we can make them visually appear as something that can be edited?
<img width="166" alt="image" src="https://github.com/intellibar/main/assets/33631005/49043dc8-34f7-4231-8102-3ea3ed9956dc">

Also I thought about the possibility to make a shortcut e.g. for copy last response
<img width="1252" alt="image" src="https://github.com/intellibar/main/assets/33631005/c95aa59d-b122-419f-81cf-31d63781cbb0">

**Reply by erusev** [reply]
Yes, we'll try to address that for 0.9.

**Comment by erusev** [comment]
You should be able to customize most shortcuts from the System Settings in macOS:

System Settings → Keyboard → Keyboard Shortcuts... → App Shortcuts

**Reply by japanese-goblinn** [reply]
Yeah, but I think that in app editor is faster and more convenient as a result
